### DISCUSSION TITLE
What is particle concentration overpotential?

### DISCUSSION THREAD

**Discussion by dion-w** [pybamm-team/PyBaMM#3554] [Q&A] [answered]
Hello everyone,
i recently updated my code to a newer PyBaMM version and saw that there is a new overpotential defined for the model called "particle concentration overpotential". I'll tell you what i understood for now, and please correct me if i'm wrong.

Before in an earlier PyBaMM version (probably 1-2 years old) i used Fickian Diffusion for solving the radial diffusion equation for the concentration in the particles.
Now, it seems to me that you're also solving for the potential in the particles due to the [MSMR formulation](https://iopscience.iop.org/article/10.1149/2.0771816jes). There the typical fickian diffusion is used in a broader sense with the chemical potential instead of the concentration. So in the msmr formulation we take the gradient of the chemical potential instead of the gradient of the concentrations. For our reactions the gradient in the chemical potential can then be written as a derivative of the open circuit potential. Therefore we need to solve for the potential in the particles but also for the concentrations.
Is this correct so far?

If yes, then i have another question. I simulate all the time with the typical DFN-model. This includes Fickian-Diffusion in the particle submodel. I even checked it the model class under model.sub_models. I still get a particle concentration overpotential if i plot it with "plot_voltage_components.py". In my assumption this should only happen if i use the MSMR submodel. Am i missing something here?

**Comment by rtimms** [comment, marked as answer]
By default, we are still using Fickian diffusion where the driving force is gradients in concentration. You have to enable the various MSMR options to use that formulation.  In MSMR you just solve for the potential, then you have a function that maps potential to stoichiometry(and therefore concentration).

In all cases, we use "particle concentration overpotential" to refer to the difference in the open-circuit potential evaluated using the bulk and surface concentrations respectively, i.e. `U_surf = U_bulk + eta_conc`. At equilibrium, the concentration gradients have relaxed so `eta_conc=0` and `U_surf = U_bulk`. In older versions of PyBaMM, we didn't explicitly include this as an overpotential, it was just tied up in the open-circuit potential.
